force-app/main/default/classes/extentionController.cls

```diff
@@ -0,0 +1,71 @@
+public class ExtentionController {
+    
+private Account acct;
+public List<contact> extContacts{get;set;}
+public String searchText{get
+    {if(searchText == null) searchText = ''; return searchText;}
+        set;}
+
+	//Constructing extension controller for standard account controller
+    public ExtentionController(ApexPages.StandardController stdCont){
+        this.acct = (Account)stdCont.getRecord();
+        try{
+        this.extContacts = [SELECT id, LastName, Title, Phone, Is_Primary_Contact__c FROM contact WHERE accountid = :acct.id ORDER BY LastName LIMIT 990];
```

Comment:
this query looks quite similar to the one in "search" method. Could you refactor your code to use 1 method for 2 queries or even to use 1 single query in 1 method just called twice?

Reply:
Created a new method. Used it in constructor, and in Search Method.